vllm/block.py

```diff
@@ -1,12 +1,43 @@
 """Token blocks."""
-from typing import List
+import weakref
+from collections import defaultdict
+from typing import Dict, List
 
 from vllm.utils import Device
 
 _BLANK_TOKEN_ID = -1
 
 DEFAULT_LAST_ACCESSED_TIME = -1
 
+TokensBlock = List[int]
+
+
+class BlockPool:
+    """A pool of physical blocks.
```

Comment:
A pool of logical blocks?

Reply:
you are right, it is a typo.

Reply:
opened https://github.com/vllm-project/vllm/pull/5620